tests/src/test/java/org/eclipse/hono/tests/client/HonoKafkaConsumerIT.java

```diff
@@ -482,7 +498,7 @@ public void testConsumerReadsLatestRecordsPublishedAfterTopicSubscriptionConfirm
      * @param ctx The vert.x test context.
      * @throws InterruptedException if test execution gets interrupted.
      */
-    @ParameterizedTest
+    @ParameterizedTest(name = IntegrationTestSupport.PARAMETERIZED_TEST_NAME_PATTERN)
     @MethodSource("partitionAssignmentStrategies")
     @Timeout(value = 10, timeUnit = TimeUnit.SECONDS)
     public void testConsumerReadsAllRecordsForDynamicallyCreatedTopics(
```

Comment:
`throws InterruptedException` can be removed here - that exception is never thrown.